tests/migration_test.go

```diff
@@ -471,6 +472,67 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 				tests.WaitForVirtualMachineToDisappearWithTimeout(vmi, 240)
 			})
 		})
+		Context("with setting guest time", func() {
+			It("should set an updated time after a migration", func() {
+				vmi := tests.NewRandomFedoraVMIWitGuestAgent()
+				vmi.Spec.Domain.Resources.Requests[k8sv1.ResourceMemory] = resource.MustParse(fedoraVMSize)
+				vmi.Spec.Domain.Devices.Rng = &v1.Rng{}
+
+				By("Starting the VirtualMachineInstance")
+				vmi = runVMIAndExpectLaunch(vmi, 240)
+
+				By("Checking that the VirtualMachineInstance console has expected output")
+				expecter, expecterErr := tests.LoggedInFedoraExpecter(vmi)
+				Expect(expecterErr).ToNot(HaveOccurred())
+				defer expecter.Close()
+
+				// Need to wait for cloud init to finnish and start the agent inside the vmi.
+				tests.WaitAgentConnected(virtClient, vmi)
+
+				By("Set wrong time on the guest")
+				_, err = expecter.ExpectBatch([]expect.Batcher{
+					&expect.BSnd{S: "date +%T -s 23:26:00\n"},
+				}, 15*time.Second)
+				Expect(err).ToNot(HaveOccurred(), "should set guest time")
```

Comment:
We may want to close the expecter connection after this line. You could wrap the part  in an anonymous function for instance. However not harmful, since the connection should break automatically with the migration anyway.